iina/HistoryController.swift

```diff
@@ -24,15 +24,26 @@ class HistoryController: NSObject {
   }
 
   private func read() {
-    history = (NSKeyedUnarchiver.unarchiveObject(withFile: plistURL.path) as? [PlaybackHistory]) ?? []
+    // Avoid logging a scary error if the file does not exist.
+    guard FileManager.default.fileExists(atPath: plistURL.path) else { return }
+    do {
+      let data = try Data(contentsOf: plistURL)
+      let object = try NSKeyedUnarchiver.unarchivedObject(ofClasses: [NSArray.self, PlaybackHistory.self],
+                                                          from: data)
+      history = object as? [PlaybackHistory] ?? []
```

Comment:
I think we should throw an error there if `as?` fails so that we can log the type casting failed, tho I don't think that will happen very often...